### PR TITLE
Update monitoring integration and backup tests

### PR DESCRIPTION
**Pull request by suleymanakbas91**
**Description**

Changes proposed in this pull request:

- Update monitoring integration and backup tests
- Enable the tests

**Related issue(s)**

See https://github.com/kyma-project/kyma/issues/6124

### CODE REVIEW

File: tests/end-to-end/backup-restore-test/backupe2e/prometheus.go [hunk 1]
```diff
@@ -202,10 +212,7 @@ func (pt *prometheusTest) testQueryTargets(url string) {
 }
 
 func isHealthy(activeTarget activeTarget) bool {
-	if activeTarget.Health == "up" {
```
Comment (clebs) [inline tests/end-to-end/backup-restore-test/backupe2e/prometheus.go]: Thanks for fixing this.

File: tests/integration/monitoring/test-monitoring.go [hunk 2]
```diff
@@ -82,11 +58,81 @@ func main() {
 	log.Printf("Monitoring tests are successful!")
 }
 
-func testGrafanaIsReady(url string) {
-	if b, statusCode := doGet(url); statusCode != 302 {
-		log.Fatalf("Test grafana: Expected HTTP response code 302 but got %v.\nResponse body: %s", statusCode, b)
+func testPodsAreReady() {
```
Comment (clebs) [inline tests/integration/monitoring/test-monitoring.go]: Awesome to finally have this! 👍

File: tests/end-to-end/backup-restore-test/backupe2e/prometheus.go (hunk 1 above)
Comment (suleymanakbas91) [reply tests/end-to-end/backup-restore-test/backupe2e/prometheus.go]: 😄

**Pull request merged**